packages/api/src/index.ts

```diff
@@ -1,17 +1,10 @@
-import * as dotenv from 'dotenv';
+import './loadEnv';
 import 'express-async-errors';
 import logger from 'jet-logger';
-import path from 'path';
 import { app } from './server';
 
-const load = dotenv.config({ path: path.join(__dirname, '..', '/.env') });
-if (load.error) {
-  logger.err(load.error.message);
-  throw load.error;
-}
-
 // Start the server
-const port = Number(process.env.PORT || 3000);
+const port = 8088;

```

Comment:
Same note as above -- I really don't like overriding env settings with static values, conditionally.

Reply:
One of my concerns was making switching between the three modes of running as easy as possible:
1. Docker - production
2. Docker - development
3. No docker - development

I'll try to add PORT support back in.

Reply:
I was thinking another option would be to replace the mongo config variables with a single MONGO_URI.

Reply:
I was thinking another option would be to replace the mongo config variables with a single MONGO_URI.

Reply:
Above is not for mongo but the api port. But yeah, perhaps we'll switch to a single MONGO_URI as well.